app/helpers/buyers_helpers.py

```diff
@@ -25,6 +26,9 @@ def is_brief_correct(brief, framework_slug, lot_slug, current_user_id):
 
 
 def is_brief_associated_with_user(brief, current_user_id):
+    current_user = data_api_client.get_user(current_user_id).get('users')
```

Comment:
isn't this in `flask-login.current_user` ?

Reply:
true, updated